es6-template-literals/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Javascript ES6 - Template Literals</title>
+  </head>
+  <body>
+    <script>
+      const number = 9200;
+      const street = 'Irvine Center Drive';
+      const address = `${number} + ' ' + ${street}`;
```

Comment:
@hollyphung95 Looks like you missed replacing some concatenation (`+`) operations:

---

![image](https://github.com/hollyphung95/m2-0723-code-solutions/assets/16143771/7dbbd615-433f-42de-a06f-8003dc95c897)

Reply:
<img width="870" alt="Screenshot 2023-07-13 at 9 28 01 PM" src="https://github.com/hollyphung95/m2-0723-code-solutions/assets/59667095/9672c11d-5af6-4874-8e2b-846b4e8864a4">